bin/http-server

```diff
@@ -68,6 +70,14 @@ if (!argv.s && !argv.silent) {
           date, req.method.cyan, req.url.cyan,
           req.headers['user-agent']
         );
+        if (argv.v || argv.verbose) {
```

Comment:
#131 will merge soon, be prepared to change how these are accessed

Reply:
Looking back at this, #131 might take a while due to extenuating circumstances, so don't worry about it!